modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/util/SyncmlGenerator.java

```diff
@@ -42,7 +42,7 @@
 
     private static Log log = LogFactory.getLog(SyncmlGenerator.class);
 
-    public String generatePayload(SyncmlDocument syncmlDocument) throws WindowsOperationException {
+    public String generatePayload(SyncmlDocument syncmlDocument) throws SyncmlOperationException {
         Document doc = generateDocument();
         Element rootElement = createRootElement(doc);
         SyncmlHeader header = syncmlDocument.getHeader();
```

Comment:
in earlier checked those things and throw runtime exceptions.so no need to handle it on this level also.